pdb2pqr/structures.py

```diff
@@ -136,10 +145,11 @@ def __init__(self, atom=None, type_="ATOM", residue=None):
             self.element = atom.element
             self.charge = atom.charge
             self.residue = residue
-            try:
-                self.mol2charge = atom.mol2charge
-            except AttributeError:
+            if isinstance(atom, ATOM):
```

Comment:
Sorry, I missed this comment. Well, according to what type checker reports and from all the definitions I saw, this should be correct. This is why type checker is important.

But it doesn't mean it's 100% guaranteed, because if you did alter a class in any weird way (which is discouraged), then it might fail. However, we should try to make it more type complete and bug-free, instead of avoiding it.